README.md

```diff
@@ -200,11 +200,30 @@ ResponseCache::forget('/some-uri');
 // Forget several URIs
 ResponseCache::forget(['/some-uri', '/other-uri']);
 
+// Using Tags (the cache driver you configured must supports tags)
+ResponseCache::forget(['/some-uri', '/other-uri'], ['tag1', 'tag2']);
+
 // Alternatively
 ResponseCache::forget('/some-uri', '/other-uri');
+
+
+// Advanced
```

Comment:
Instead of a single word comment, close the code block, add a paragraph describing the next example, and open a new code block.

Reply:
ok